test/integration/reasoner/ExplanationTest.java

```diff
@@ -488,10 +488,10 @@ public void test_nested_explanations() {
 
     private List<Explanation> assertSingleExplainableExplanations(ConceptMap ans, int anonymousConcepts, int explainablesCount,
                                                                   int explanationsCount, RocksTransaction txn) {
-        List<ConceptMap.Explainables.Explainable> explainables = ans.explainables().explainables().toList();
+        List<ConceptMap.Explainable> explainables = ans.explainables().explainables().toList();
         assertEquals(anonymousConcepts, iterate(ans.concepts().keySet()).filter(Identifier::isAnonymous).count());
         assertEquals(explainablesCount, explainables.size());
-        ConceptMap.Explainables.Explainable explainable = explainables.iterator().next();
+        ConceptMap.Explainable explainable = explainables.iterator().next();
```

Comment:
Maybe just take the element at index 0?
```suggestion
        ConceptMap.Explainable explainable = explainables.get(0);
```

Reply:
good catch